sdk/iOS/src/MSPullSettings.h

```diff
@@ -0,0 +1,20 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import <Foundation/Foundation.h>
+
+/// Settings that control the pull behavior
+@interface MSPullSettings : NSObject
+
+#pragma mark * Properties
+
+///@name Properties that control pull behavior
+///@{
+
+/// Size of pages requested from the server while performing a pull
+@property (nonatomic) NSInteger pageSize;
+
+///@}
+
```

Comment:
Lets do it in a separate log, I'm not sure we'd want the default to be YES on this either.